frontend/src/views/Org/AuditLogsPage/components/LogsTableRow.tsx

```diff
@@ -476,7 +478,47 @@ export const LogsTableRow = ({ auditLog, isOrgAuditLogs, showActorColumn }: Prop
             </Tooltip>
           </Td>
         );
+
+      case EventType.GET_WORKSPACE_KEY:
+        return (
+          <Td>
+            <p>{`Key ID: ${event.metadata.keyId}`}</p>
+          </Td>
+        );
+
+      case EventType.LOGIN_IDENTITY_UNIVERSAL_AUTH:
```

Comment:
Note: It would be great to spend a day (I can do it if needed) to make sure that all audit log events are being shown in this table (including other logins across auth methods).

No action item is needed for this review comment.